LibrarySystemBackend/LibrarySystem/LibrarySystem/Services/BookService.cs

```diff
@@ -113,7 +117,18 @@ public async Task<bool> RemoveBookAsync(int bookId)
                 throw ex;
             }
         }
-       
+        public async Task<List<BookDetails>> GetBookDetailsAsync()
```

Comment:
Add try catches and summaries to new methods

Reply:
Done